packages/blast-tests/integration-tests/vars.sh

```diff
@@ -28,64 +28,4 @@ blast.config.js
 contracts
 integration_tests
 package.json
-scripts
-
-./contracts:
-alpha
-beta
-
-./contracts/alpha:
-Cargo.toml
-examples
-rustfmt.toml
-schema
-src
-
-./contracts/alpha/examples:
-schema.rs
-
-./contracts/alpha/schema:
-count_response.json
-execute_msg.json
-instantiate_msg.json
-query_msg.json
-state.json
-
-./contracts/alpha/src:
-contract.rs
-error.rs
-lib.rs
-msg.rs
-state.rs
-
-./contracts/beta:
-Cargo.lock
-Cargo.toml
-examples
-rustfmt.toml
-schema
-src
-
-./contracts/beta/examples:
-schema.rs
-
-./contracts/beta/schema:
-count_response.json
-execute_msg.json
-instantiate_msg.json
-query_msg.json
-state.json
-
-./contracts/beta/src:
-contract.rs
-error.rs
-lib.rs
-msg.rs
-state.rs
-
-./integration_tests:
-alpha.test.js
-
-./scripts:
-deploy.js
-interact.js'
+scripts'
```

Comment:
Here you are removing recursive folder/file check. During past development I had a bug with recursive file copy so it is a good idea to add a check for at least 1 subfolder/file. If you don't feel like doing it in this task, we can make another.